### PR TITLE
Add log scaling to equity plot and coin plots

### PR DESCRIPTION
**Pull request by sheridavandenbent**
Adds log scaling to the equity plot and the coin price plots. Log scaling is default on for the equity plot, and default off for the coin price plots.

Also removes a (now) redundant check to remove old plots in an old folder.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
   "strategy-name": "MyStrategy",
   "strategies-folder": "resources/setup/strategies",
   "plots": true,
+  "plot_log_scale": {
+    "equity": true,
+    "per_coin": false
+  },
```
Comment (marijn111) [inline config.json:26]: Can't we switch to the log scale within the plot? That would be the ideal implementation.
https://community.plotly.com/t/set-linear-or-log-axes-from-button-or-drop-down-menu-python/34927

Comment (sheridavandenbent) [reply config.json]: Done! Thanks for the great suggestion!

**Comment by marijn111** [comment]
This doesn't seem like it is working correctly just yet. Very zoomed out and the colouring is inverted when going negative. 

newplot

File: modules/output/equity_plot.py [hunk 2]
```diff
@@ -21,5 +22,19 @@ def equity_plot(capital_dict):
     # create figure
     fig = go.Figure()
     fig.add_trace(go.Scatter(x=dates, y=df[1], fill='tozeroy'))  # fill down to xaxis
-    fig.update_yaxes(range=[min_value, max_value])
```
Comment (marijn111) [inline modules/output/equity_plot.py:24]: Could you re-add this line?

**Review by marijn111 — CHANGES_REQUESTED** [review]

File: modules/output/equity_plot.py (hunk 2 above)
Comment (sheridavandenbent) [reply modules/output/equity_plot.py:24]: Added this functionality in a slightly different way!

**Review by marijn111 — APPROVED** [review]

**Pull request merged**